go/build.sh

```diff
@@ -27,14 +27,16 @@ if [ "${core_lib_path}" = "" ]; then
 	usage "path to core lib not specified"
 fi
 
+github_lib_path="${4-"github.com/ionos-cloud/sdk-go/v5"}"
```

Comment:
you're missing a `:` - it should read:

`github_lib_path="${4:-"github.com/ionos-cloud/sdk-go/v5"}"`